src/sentry/templates/sentry/feedback.txt

```diff
@@ -0,0 +1,41 @@
+{% spaceless %}
+{% autoescape off %}
+{% if enhanced_privacy %}
+Details about this issue are not shown in this notification since enhanced
+privacy controls are enabled. For more details about this issue, view this
+issue on Sentry.
```

Comment:
```suggestion
Details about this feedback are not shown in this notification since enhanced
privacy controls are enabled. For more details about this feedback, view this
issue on Sentry.
```